### DISCUSSION TITLE
[Bounty] $10 to He Who Adds Pixel Tilting

### DISCUSSION THREAD

**Discussion by ReturnToZender** [Bubberstation/Bubberstation#1803] [ideas/bounties]
Pixel tilting is where a button is used to tilt the character. The proc the game uses for something like this is transform.tilt, I _believe,_ but that might have changed on TG since. If it did, I believe it may be transform.turn instead.

Original PR where it was made:
https://github.com/SPLURT-Station/S.P.L.U.R.T-Station-13/pull/12

**Comment by Odairu** [comment]
This is completed and merged

**Comment by ReturnToZender** [comment]
Complete and merged
